### PR TITLE
feat: Add feature list to organization serializer

### PR DESCRIPTION
**Pull request by lynnagara**
This change returns the features list for the organization index endpoint.
Since the home route for each organization now depends on which features
are active, we need to return the list of features in order to make the
organization switcher work.

### CODE REVIEW

File: src/sentry/api/serializers/models/organization.py [hunk 1]
```diff
@@ -53,6 +56,37 @@ def serialize(self, obj, attrs, user):
...
+        # Retrieve all registered organization features
+        org_features = features.all(feature_type=OrganizationFeature).keys()
+        feature_list = set()
+
+        for feature_name in org_features:
+            if not feature_name.startswith('organizations:'):
+                continue
+            if features.has(feature_name, obj, actor=user):
+                # Remove the organization scope prefix
+                feature_list.add(feature_name[len('organizations:'):])
```
Comment (markstory) [inline src/sentry/api/serializers/models/organization.py]: You call `len()` multiple times on a fixed string. You could hoist that above the loop to save some time. :golf:

**Review by markstory — COMMENTED** [review]
Should we add an endpoint test or a serializer test as we are adding to a public api?

File: src/sentry/api/serializers/models/organization.py (hunk 1 above)
Comment (lynnagara) [reply src/sentry/api/serializers/models/organization.py]: On second thoughts, will prefer to not touch this for now

Comment (EvanPurkhiser) [reply src/sentry/api/serializers/models/organization.py]: Yeah this was my doing 🤷‍♂️

**Review by markstory — APPROVED** [review]

**Pull request merged**